ipn/localapi/localapi.go

```diff
@@ -1037,6 +1038,69 @@ func (h *Handler) serveTKAModify(w http.ResponseWriter, r *http.Request) {
 	w.Write(j)
 }
 
+// serveProfiles serves profile switching-related endpoints. Supported methods
+// and paths are:
+// - GET /profiles/: list all profiles (JSON-encoded array of ipn.LoginProfiles)
+// - PUT /profiles/: add new profile (no response)
+// - GET /profiles/current: current profile (JSON-ecoded ipn.LoginProfile)
```

Comment:
What about `GET /profiles/<id>`? Not listed here.

Reply:
There's no `LocalBackend` method to get a specific profile. I guess I could do it by iterating over the return value of `ListProfiles()`, but so far we have not needed it.

Reply:
Added support for `GET /profiles/<id>` too.